docs/testplan/GCU-Dynamic-ACL-testplan.md

```diff
@@ -137,15 +137,53 @@ Verify that after creation, a forward ACL rule can be removed and packets matchi
 - Verify that the results of "show acl rule {rule_name}" are empty for both rule names
 
 - Verify that packets with IPs matching the removed forwarding rules are dropped on this specific port
+- 
+### Test Case # 7 - DHCP Forwarding Rule
 
-### Test Case # 7 - Scale test of ACL Table, add large amount of forward and drop rules
+#### Test Objective
+
+Verify that GCU is capable of adding a blanket DHCP forwarding rule, and that traffic follows these rules accordingly
+
+#### Testing Steps
+
+- Use GCU to create a blanket DHCP packet forwarding rule
+
+- Use GCU to create a blanket drop rule on a port
+
+- Verify that all operations were successful
+
+- Verify that DHCP packets are forwarded
+
+- Verify that all other packets on this port are dropped
+
+### Test Case # 8 - ARP Forwarding Rule
+
+#### Test Objective
+
+Test that GCU is capable of adding a blanket ARP forwarding rule, and that traffic follows this rule accordingly
+
+#### Testing Steps
+
+- Use GCU to create a blanket ARP packet forwarding rule
+
+- Use GCU to create a blanket drop rule on a port
+
+- Verify that all operations were successful
+
+- Verify that ARP packets are forwarded
```

Comment:
The ARP packet will not be forwarded by T0. The T0 will trap the ARP packet. So we can send out ARP request and verify the ARP reply from server.